### ISSUE TITLE
Access columns via URL?

### ISSUE THREAD

**Issue by DickenSerm** [glanceapp/glance#162]
Would it be easy to implement a URL parameter like https://glance.url/home?column=left

**Comment by svilenmarkov** [comment]
Would this just default to showing that column first? Currently on mobile it defaults to showing the full column.

**Comment by DickenSerm** [comment]
Exactly, I don't know much about programming, if it's just a small thing, I would like to have the option to save it as a link on the smartphone.
